tasks/customaction.py

```diff
@@ -23,7 +23,7 @@
 AGENT_TAG = "datadog/agent:master"
 
 @task
-def build(ctx, vstudio_root=None, arch="x64"):
+def build(ctx, vstudio_root=None, arch="x64", debug=None):
```

Comment:
if `debug` is meant to be a boolean, it would make more sense to set its default value to `False`